lib/redis_memo/memoize_query/cached_select.rb

```diff
@@ -368,6 +376,17 @@ def self.===(node)
     end
   end
 
+  class NodeHasComparators
+    def self.===(node)
+      case node
+      when Arel::Nodes::LessThan, Arel::Nodes::LessThanOrEqual, Arel::Nodes::GreaterThan, Arel::Nodes::GreaterThanOrEqual, Arel::Nodes::NotEqual
```

Comment:
Wouldn't NotEqual already be covered [here](https://github.com/chanzuckerberg/redis-memo/blob/main/lib/redis_memo/memoize_query/cached_select.rb#L340)? If so, then we can probably remove the condition with just `Arel::Nodes::NotEqual` if we have this condition

Reply:
oh good catch thanks! I forgot to mention that line 341 can be removed after this will be merged.

Reply:
Why do we need to wait for this to be merged in order to remove the code on line 341? Couldn't we remove it as part of this PR?

Reply:
Yup I missed this option, thanks : )